packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -1203,6 +1203,7 @@ export class AvailableSlotsService {
       frequency: eventType.slotInterval || input.duration || eventType.length,
       datesOutOfOffice: !isTeamEvent ? allUsersAvailability[0]?.datesOutOfOffice : undefined,
       showOptimizedSlots: eventType.showOptimizedSlots,
+      datesOutOfOfficeTimeZone: !isTeamEvent ? allUsersAvailability[0]?.timeZone : undefined,
```

Comment:
nit: Might be a good idea to refactor datesOutOfOffice to be an object that could hold both dates and timezone. Good for followup maybe